README.md

```diff
@@ -37,9 +37,9 @@ You can run CodiMD in a number of ways, and we created setup instructions for
 all of these:
 
 * [Docker](docs/setup/docker.md)
-* [Docker (LinuxServer.io)](docs/setup/docker-linuxserver.md)
 * [Kubernetes](docs/setup/kubernetes.md)
 * [Cloudron](docs/setup/cloudron.md)
+* [LinuxServer.io (container-based)](docs/setup/docker-linuxserver.md)
```

Comment:
> [LinuxServer.io (container-based)](docs/setup/docker-linuxserver.md)

should probably read

> [LinuxServer.io (Docker Image)](docs/setup/docker-linuxserver.md)

Reply:
There is mainly one reason why I try to avoid the word "Docker" in here: https://www.youtube.com/watch?v=PQaz2PDW4MM&t=1m29s (also the reason why the repository is called container, not docker)